diff --git a/colors/dracula.vim b/colors/dracula.vim
index 14f6542..77ead01 100644
--- a/colors/dracula.vim
+++ b/colors/dracula.vim
@@ -200,12 +200,16 @@ set background=dark
 
 " Required as some plugins will overwrite
 call s:h('Normal', s:fg, g:dracula_colorterm || has('gui_running') ? s:bg : s:none )
-call s:h('StatusLine', s:none, s:bglighter, [s:attrs.bold])
-call s:h('StatusLineNC', s:none, s:bglight)
-call s:h('StatusLineTerm', s:none, s:bglighter, [s:attrs.bold])
-call s:h('StatusLineTermNC', s:none, s:bglight)
+" call s:h('StatusLine', s:none, s:bglighter, [s:attrs.bold])
+hi! StatusLine none ctermfg=15
+" call s:h('StatusLineNC', s:none, s:bglight)
+hi! StatusLineNC none ctermfg=07
+" call s:h('StatusLineTerm', s:none, s:bglighter, [s:attrs.bold])
+hi! StatusLineTerm none ctermfg=15
+" call s:h('StatusLineTermNC', s:none, s:bglight)
+hi! StatusLineTermNC none ctermfg=07
 call s:h('WildMenu', s:bg, s:purple, [s:attrs.bold])
-call s:h('CursorLine', s:none, s:subtle)
+" call s:h('CursorLine', s:none, s:subtle)
 
 hi! link ColorColumn  DraculaBgDark
 hi! link CursorColumn CursorLine
